Sources/EpoxyCore/SwiftUI/LayoutUtilities/MeasuringViewRepresentable.swift

```diff
@@ -112,14 +110,38 @@ extension MeasuringViewRepresentable {
     -> CGSize?
   {
     nsView.strategy = sizing
-
-    // Creates a size by replacing `nil`s with `UIView.noIntrinsicMetric`
-    nsView.proposedSize = .init(
-      width: proposal.width ?? ViewType.noIntrinsicMetric,
-      height: proposal.height ?? ViewType.noIntrinsicMetric)
-
+    nsView.proposedSize = proposal.viewTypeValue
     return nsView.measuredFittingSize
   }
   #endif
 }
 #endif
+
+#if swift(>=5.7.1) // Proxy check for being built with the iOS 15 SDK
+@available(iOS 16.0, tvOS 16.0, macOS 13.0, *)
+extension ProposedViewSize {
+  /// Creates a size by capping infinite values to a significantly large value and replacing `nil`s
+  /// with `UIView.noIntrinsicMetric`
+  var viewTypeValue: CGSize {
```

Comment:
I don't really understand the "viewType" part of this name

Reply:
Added a comment to clarify